FilterViewController.m

```diff
@@ -27,11 +27,11 @@ - (void)viewDidLoad {
     MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
     
     //sends double array for filternames and single array for header lines
-    [labels setDataWithFilterNames:@[self.returnLabelArray] headerTitles:@[@"Labels"]];
+    [labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
```

Comment:
Sorry I lead you astray on this one. I didn't read that it was already an array and didn't need to be wrapped in an array